src/app/components/feed/FeedItem.module.css

```diff
@@ -72,7 +72,7 @@
   gap: 8px;
 }
 
-.feedItemTeamCardFooter > div > div {
+.feedItemTeamCardFooter > div {
   flex-wrap: nowrap !important;
```

Comment:
reviewer note, i removed a wrapper div in the Bullet Separator component, so this looks one level less to find it